### PR TITLE
added tag for fluentd on instance type entity

### PR DESCRIPTION
**Pull request by vibhory2j**
added tag for fluentd on instance type entity

### CODE REVIEW

**Comment by vibhory2j** [comment]
Tested in controlled environment and works fine. Next step is to write the unit test for the new method added.

**Comment by mohabusama** [comment]
Assuming extending taupage/stups support, such info should be retrievable from the User data of the instance. Similar to this line [here](https://github.com/zalando-zmon/zmon-aws-agent/blob/master/zmon_aws_agent/aws.py#L250) where we retrieve ``application_id``, ``stack_version`` etc...

File: tests/conftest.py [hunk 1]
```diff
@@ -261,6 +261,10 @@ def get_apps():
             'application_id': 'app-1', 'source': 'registry/stups/zmon-aws-agent:cd81',
             'ports': [2222], 'runtime': 'docker',
             'application_version': '1.0',
+            'logging': {
+                'fluentd_enabled': 'true',
```
Comment (mohabusama) [inline tests/conftest.py]: `s/true/True`

**Comment by mohabusama** [comment]
👍

**Comment by vibhory2j** [comment]
👍

**Pull request merged**